src/test/TestArgs.cpp

```diff
@@ -80,6 +80,16 @@ R"(
     "addons",
     llvm::cl::desc("The directory with the addons."),
     llvm::cl::cat(commonCat))
+
+, systemIncludes(
+    "system-includes",
+    llvm::cl::desc("A list of paths to the standard library."),
```

Comment:
Isn't system-includes corresponding to `-isystem`? 

- If so, it's "A list of paths to the system headers.". 
- If not, then the `isystem` option is missing.